toxav/bwcontroller.c

```diff
@@ -186,6 +183,54 @@ static int on_update(BWController *bwc, const struct BWCMessage *msg)
     return 0;
 }
 
+/* !!hack!! TODO:fix me */
```

Comment:
I'd move these declarations to a new header, maybe `toxav_hacks.h`.

Reply:
done